specifications/transactions/travel-rule/README.md

```diff
@@ -0,0 +1,372 @@
+| `place_of_birth`    | `AddressObject`    | N        | Place of birth for this user. line1 and line2 fields should not be populated for this usage of the address Object |
+| `national_id`       | `NationalIdObject` | N        | National ID information for the holder of this account |
+| `legal_entity_name` | `str`              | N        | Name of the legal entity. Used when subaddress represents a legal entity rather than an individual. KYCDataObject should only include one of legal_entity_name OR given_name/surname |
+
+```
+{
+    "payload_version": 1,
+    "type": "individual",
+    "given_name": "ben",
+    "surname": "maurer",
+    "address": {
+        AddressObject(),
+    },
+    "dob": "1920-03-20",
+    "place_of_birth": {
+        AddressObject(),
+    }
+    "national_id": {
+    },
+}
+```
+
+### AddressObject
+
+Represents a physical address
+
+| Field         | Type  | Required | Description |
+|-------        |------ |----------|-------------|
+| `city`        | `str` | N        | The city, district, suburb, town, or village |
+| `country`     | `str` | N        | [Two-letter country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)|
+| `line1`       | `str` | N        | Address line 1|
+| `line2`       | `str` | N        | Address line 2 - apartment, unit, etc. |
+| `postal_code` | `str` | N        | ZIP or postal code |
+| `state`       | `str` | N        | State, county, province, region. |
+
+```
+{
+    "city": "Sunnyvale",
+    "country": "US",
+    "line1": "1234 Maple Street",
+    "line2": "Apartment 123",
+    "postal_code": "12345",
+    "state": "California",
+}
+```
+
+### NationalIdObject
+
+Represents a national ID.
+
+| Field      | Type  | Required | Description |
+|-------     |------ |----------|-------------|
+| `id_value` | `str` | Y        | Indicates the national ID value,for example, a social security number |
+| `country`  | `str` | N        | [Two-letter country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) |
+| `type`     | `str `| N        | Indicates the type of the ID |
+
+```
+{
+    "id_value": "123-45-6789",
+    "country": "US",
+    "type": "SSN",
+}
+```
+
+### PaymentActionObject
+
+| Field       | Type   | Required | Description |
+|-------      |------  |----------|-------------|
+| `amount`    | `uint` | Y        | Amount of the transfer as defined by the on-chain base units. For example, if XUS is represented on-chain as "1" equals 1e-6 units, then "1" here would also equal 1e-6 units. |
+| `currency`  | `enum` | Y        | One of the supported currency types, for example, XUS |
+| `action`    | `enum` | Y        | Populated in the request. This value indicates the requested action to perform, and the only valid value is charge. |
+| `timestamp` | `uint` | Y        | Unix time indicating the time that the payment `Command` was created. |
```

Comment:
This wasn't obvious to me how timestamp is used by the sender/receiver.  Just for record keeping?

Reply:
@xli ?